### PR TITLE
K6 test scripts

### PR DESCRIPTION
**Pull request by loneil**
# Description
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-2507

A parameterized K6 test to do **public** form load, submit, success flow (so a public submission as a user would do it) through CHEFS API calls.

See READMEs in this PR for details.

## Types of changes

New feature (non-breaking change which adds functionality)
Documentation (non-breaking change with enhancements to documentation)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

Don't necessarily have to push this through CHEFS env after merge. No actual CHEFS code changed here.

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-289

File: tests/load/k6/public-form-submit/script.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+// 2. Setup
+// -------------------------------------------------------------------------------------------------
+export function setup() {
+  // Not much needed in setup in this script. Using the new k6 SharedArray construct above to load the file
+  console.log(`CHEFS url: ${chefsUrl}`);
+  console.log(`Form Id: ${formId}`);
+  console.log(`Form Type: ${formType}`);
+
+  // Check a formId guid was supplied or nothing will work
+  var pattern = new RegExp('^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$', 'i');
```
Comment (jujaga) [inline tests/load/k6/public-form-submit/script.js:64]: const

**Review by jujaga — COMMENTED** [review]
Looks pretty cool. I think we should move all the example json files to `tests/load/k6/public-form-submit/fixtures` as keeping all test data constructs like these JSON files makes more sense grouped together as explicit fixtures.

File: tests/load/k6/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# CHEFS K6 Tests
+
+The K6 scripts contained here are for load testing purposes in CHEFS.
+
+If you are not familiar with K6, please start with the documentation on [their site](https://k6.io/docs/#what-is-k6).
+
+It is important not to run load test scripts against production (or other imporant environments) unless controlled. As well, if load tests are ever being run against an OpenShift hosted CHEFS instance using a very high volume (in the thousands of requests or higher) you should check with platform services as high network incoming traffic should be monitored and done outside peak times.
```
Comment (TimCsaky) [inline tests/load/k6/README.md:7]: imporant > important

Comment (loneil) [reply tests/load/k6/README.md:7]: fixed

File: tests/load/k6/public-form-submit/script.js (hunk 1 above)
Comment (loneil) [reply tests/load/k6/public-form-submit/script.js:64]: fixed

**Review by TimCsaky — APPROVED** [review]
nicely done. We should use this to test CHEFS this on openshift. 
Be interesting to see how the platform, app, database etc cope.

**Review by jujaga — APPROVED** [review]

**Pull request merged**